cmd/migrate_storage.go

```diff
@@ -25,13 +27,13 @@ import (
 var CmdMigrateStorage = cli.Command{
 	Name:        "migrate-storage",
 	Usage:       "Migrate the storage",
-	Description: "This is a command for migrating storage.",
+	Description: "This is a command for migrating Gitea configured storage to a parameter indicated storage.",
```

Comment:
OK. I agree with others in that this is difficult to read.

As an initial comment, I think we should probably look at the `Usage` string too - I'm not too keen on the `the` but I see it matches the `migrate` (database) `Usage` and I guess a lot of the `Usage` and `Description` strings need some work to make them consistent and more idiomatic.

However, let's just consider this sub-command and be holistic about it.

### What does this command do?

A somewhat verbose and poor description would be: This command first of all reads the app.ini file to work out how `Storage` is currently configured and then **copies/converts/transfers** some subset of all those "stored files" to another `Storage` type which has been configured using options on the command-line.

Looking at the action and the object here we're looking at a command that is COPYing STORAGE or CONVERTing STORAGE.

### Is `migrate-storage` even the right name for this sub-command?

The above reasoning suggests that this command is not named correctly. 

To further illustrate this point: Gitea has at least 2 other meanings of the term `migrate` and this adds a third.

Migrate means:

1. Update DB schema (this was the initial meaning) OR
2. Clone a repository from a different server, either just the git files but possibly also the other data too. (This now appears to be the dominant meaning.)

This command adds a third meaning:

3. Copy data from one storage type to another.

I think therefore this is badly named and it should be `copy-storage`, `convert-storage`, `transfer-storage`, `import-storage` or the like. 

We have to get better at not overloading concepts - it's confusing for everyone.

I think therefore this sub-command needs renaming to something like I've suggested above. 

ASIDE: Similarly I think `gitea migrate` will need to be changed too. (Likely to something like `gitea update-database-schema` with all code relating to database schema updates being changed from `migration` to at least `dbmigration` if not `schemaupdate`.) 

### What should this description be? 

As I suggest above, I think the concept of `migration` is too overloaded at this point to be useful, so we should not use it in the Description even if we decide not to change the sub-command's name.

Next we should drop the `This is a command for` as it is unnecessary and most guidelines would suggest that we simply state what the command does.

These two points lead me to suggest:

```suggestion
	Description: "Copies stored files to new storage",
```

If that is too brief we could add something about how the storage is configured:

```suggestion
	Description: "Copies stored files from storage configured in app.ini to parameter-configured storage",
```